dev_guide/builds.adoc

```diff
@@ -33,17 +33,21 @@ link:#docker-strategy-options[options])
 (link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[description],
 link:#custom-strategy-options[options])
 
-And there are three types of build source:
+And there are four types of build source:
 
 - link:#source-code[Git]
 - link:#dockerfile-source[Dockerfile]
+- link:#image-source[Image]
 - link:#binary-source[Binary]
 
 It is up to each build strategy to consider or ignore a certain type of source,
 as well as to determine how it is to be used.
 
-Binary and Git are mutually exclusive source types, while Dockerfile can be used
-by itself or together with Git and Binary.
+Binary and Git are mutually exclusive source types, while Dockerfile and Image can be used
+by themselves, with each other, or together with either Git or Binary.
+
+Also, the Binary build source type is unique from the other options in how it is
+specified to the system.  Exactly how is detailed link:#binary-source[below].
```

Comment:
@rhcarvalho many of us older folks were taught to use two spaces after a period, not one.  Personally I still prefer it.  I haven't audited our docs to see if we're doing both or one exclusively (any docs i've written i expect have two)., but i'd leave it up to the docs folks to audit for stylistic decisions like that and so far they don't seem concerned.